### PR TITLE
Use PILImage.frombytes when PILImage.fromstring gives exception

### LINKED ISSUE #4710: Error: fromstring() in core/image/img_pil.py
Platform: Linux (OpenSuse, Ubuntu)

[INFO   ] [Kivy        ] v1.9.1
[INFO   ] [Python      ] v2.7.12 (default, Jul 01 2016, 15:36:53) [GCC]

Error:
  File "/usr/lib64/python2.7/site-packages/kivy/core/image/img_pil.py", line 105, in save
    image = PILImage.fromstring(fmt.upper(), (width, height), pixels)
  File "/usr/lib64/python2.7/site-packages/PIL/Image.py", line 2063, in fromstring
    "Please call frombytes() instead.")
Exception: fromstring() has been removed. Please call frombytes() instead.

In File "/usr/lib64/python2.7/site-packages/kivy/core/image/img_pil.py"
Line 105:
image = PILImage.fromstring(fmt.upper(), (width, height), pixels)

use...

image = PILImage.frombytes(fmt.upper(), (width, height), pixels)

### PR DESCRIPTION
**Pull request by udiboy1209**
Fixes #4710

Was trying to make a test for this but Image.save has some other error(#4727)

### CODE REVIEW

File: kivy/core/image/img_pil.py [hunk 1]
```diff
@@ -102,7 +102,11 @@ def load(self, filename):
 
     @staticmethod
     def save(filename, width, height, fmt, pixels, flipped=False):
-        image = PILImage.fromstring(fmt.upper(), (width, height), pixels)
+        # Support PIL 3+
+        try:
+            image = PILImage.fromstring(fmt.upper(), (width, height), pixels)
+        except:
```
Comment (akshayaurora) [inline kivy/core/image/img_pil.py]: try not to use a blanked exception. What exact error are you getting with existing code?

Comment (akshayaurora) [inline kivy/core/image/img_pil.py]: ok, on further review, from string has simply been removed from Pillow. Just replace fromstring in this case with `frombytes`. That should work with all mordern versions of Pillow.

**Comment by akshayaurora** [comment]
@udiboy1209 thanks, nice work!

**Comment by udiboy1209** [comment]
thanks goes to @steve1805 for finding and suggesting the fix!

**Pull request merged**